proto/src/main/proto/Rss.proto

```diff
@@ -590,7 +590,7 @@ message ReassignServersReponse{
 
 message RssReassignFaultyShuffleServerRequest{
   int32 shuffleId  = 1;
-  repeated string partitionIds = 2;
+  repeated int32 partitionIds = 2;
```

Comment:
Is it enough for Spark, MR and Tez?

Reply:
Spark is enough

Reply:
cc @zhengchenyu Could you ensure this point?